toptal/picasso
# chore: disallow usage of lab and forms packages inside picasso code

[FX-1916]

### Description

Lab and Forms packages are not dependencies of picasso, so picasso can't use anything from those packages without bloating the bundle.

We do not need to remove FileList from the index.js, because it's there already for many versions and breaking change is required to remove it

### How to test

- checkout to the branch locally
- try to add `import { ProgressBar } from '@toptal/picasso-lab'` inside picasso component
- make sure that linting fails for this file

### Screenshots


![image](https://user-images.githubusercontent.com/1291032/130199863-92afeabc-03d5-4f06-b10a-9fc4db5db115.png)


Before
![image](https://user-images.githubusercontent.com/1291032/130200099-a108ee0b-f22b-4d46-b506-07674887e46d.png)

After
![image](https://user-images.githubusercontent.com/1291032/130200162-0ed53342-1d23-4587-b84a-2593dd869e47.png)


### Review


<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>


[FX-1916]: https://toptal-core.atlassian.net/browse/FX-1916
